openfisca_us/variables/finance/benefit/tanf.py

```diff
@@ -38,6 +38,12 @@ class tanf_max_amount(Variable):
     documentation = "The maximum benefit amount a family could receive from Temporary Assistance for Needy Families given their state and family size."
     unit = "currency-USD"
 
+    def formula(spm_unit, period, parameters):
+        family_size = spm_unit.nb_persons()
```

Comment:
This might fix the current test failure
```suggestion
        family_size = spm_unit.nb_persons().astype(str)
```